ghost/admin/app/helpers/history-event-fetcher.js

```diff
@@ -91,11 +91,30 @@ export default class HistoryEventFetcher extends Resource {
                 this.hasReachedEnd = true;
             }
 
-            actions.forEach((a) => {
+            let count = 1;
+
+            actions.reverse().forEach((a, index) => {
+                const nextAction = actions[index + 1] || null;
+
+                // depending on the similarity, add additional properties to be used on the frontend for grouping
+                // skip - used for hiding the event on the frontend
+                // count - the number of similar events which is added to the last item
+                if (nextAction || (!nextAction && actions[index - 1].skip)) {
+                    if (nextAction && a.resource_id === nextAction.resource_id && a.event === nextAction.event) {
+                        a.skip = true;
```

Comment:
I was gonna write a comment here about how we could use "reduce" here to make things cleaner and only keep the "viewActions" without the skip property. but this approach is more readable and is "good enough" to do what we need 👌